content/applications/studio/approval_rules.rst

```diff
@@ -0,0 +1,79 @@
+==============
+Approval rules
+==============
+
+Studio approval rules are used to automate approval processes for actions. They allow you to define
+the criteria for when an approval is required before an action can be performed using a button.
+
+Configuration
+=============
+
+To add approval rules, proceed as follows:
+
+#. :ref:`Open Studio <studio/access>` and switch to the required :doc:`view <views>`.
+#. Select the button for which you want to add approval rules.
+#. In the :guilabel:`Properties` tab on the left, enable the :guilabel:`Set approval rules`
+   feature.
+#. Specify the :guilabel:`Allowed Group` to limit the approval permission to a specific user
+   group.
+#. Select the :guilabel:`Responsible` user to create an activity for a specific user when approval
+   is requested from them.
+#. Select the :guilabel:`Users to notify` via internal note.
+#. Add a :guilabel:`Description` to be displayed in the :ref:`Approval dialog <approval-rules/use>`.
+
+Optionally, you can also add conditions for the approval rule to be applied by clicking the
+:icon:`fa-filter` (:guilabel:`filter`) icon next to the :guilabel:`Allowed Group` field.
+
+Click :guilabel:`Add an approval rule` to add another rule. When there are multiple approval rules,
```

Comment:
```suggestion
To add another rule, click :guilabel:`Add an approval rule`. When there are multiple approval rules,
```
I prefer structuring the sentence this way so the reader immediately knows if they want to click this or not. It makes sense to me to put intention ahead of instruction if that makes sense :)